admin/htop/Makefile

```diff
@@ -12,8 +12,8 @@ PKG_VERSION:=3.4.1
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/htop-dev/htop/tar.gz/$(PKG_VERSION)?
-PKG_HASH:=af9ec878f831b7c27d33e775c668ec79d569aa781861c995a0fbadc1bdb666cf
+PKG_SOURCE_URL:=https://codeload.github.com/htop-dev/htop/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
```

Comment:
I think this does not belong to vim update

Reply:
Fixed